### PR TITLE
Docs/overview components

### PR DESCRIPTION
**Pull request by apenzk**
# Description of change

added overview of components, changed pollen to goshimmer, sorted files in docs

## Type of change

- Documentation Fix

## How the change has been tested

No tests since documentation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 * Fix locking management on message factory
 * Fix several visualizer bugs
 * Update Grafana pie charts
-* Update snapshot file with Pollen UTXO at 2021-05-25 16:54 UTC
+* Update snapshot file with Goshimmer UTXO at 2021-05-25 16:54 UTC
```
Comment (capossele) [inline CHANGELOG.md:9]:
```suggestion
* Update snapshot file with Pollen UTXO at 2021-05-25 16:54 UTC
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -165,7 +165,7 @@
 * Add Mana (currently not used by any of the modules)
 * Add Mana APIs
 * Add Mana section to the local dashboard
-* Add Mana section to the Pollen Analyzer dashboard
+* Add Mana section to the Goshimmer Analyzer dashboard
```
Comment (capossele) [inline CHANGELOG.md:168]:
```suggestion
* Add Mana section to the Pollen Analyzer dashboard
```

Comment (capossele) [reply CHANGELOG.md:168]: Changelogs should not be changed

File: plugins/analysis/dashboard/frontend/src/app/App.tsx [hunk 3]
```diff
@@ -23,7 +23,7 @@ class App extends React.Component<AppProps, unknown> {
                 <header>
                     <Link className="brand" to="/">
                         <img src="/assets/logo-header.svg" alt="Pollen Analyzer" />
-                        <h1>Pollen Analyzer</h1>
+                        <h1>Goshimmer Analyzer</h1>
```
Comment (capossele) [inline plugins/analysis/dashboard/frontend/src/app/App.tsx:26]:
```suggestion
                        <h1>Pollen Analyzer</h1>
```

Comment (capossele) [reply plugins/analysis/dashboard/frontend/src/app/App.tsx:26]: this one will be changed in the future

File: plugins/drng/drng.go [hunk 4]
```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	// Pollen defines the instance ID of the Pollen drng committee.
+	// Pollen defines the instance ID of the Goshimmer drng committee.
```
Comment (capossele) [inline plugins/drng/drng.go:16]: here and in general for the entire dRNG we need to think of a better name for the committee

**Review by capossele — CHANGES_REQUESTED** [review]

File: docs/SUMMARY.md [hunk 5]
```diff
@@ -26,11 +26,10 @@ http://goshimmer.docs.iota.org/
   - [Object storage](./implementation_design/object_storage.md)
 
 - [Protocol specification](./protocol_specification.md)
-  - [Protocol high level overview](./application_logic/protocol.md)
-  - [Layers](./concepts/layers.md)
-  - [Mana](./specification/001-mana.md)
-  - [Markers](./specification/003-markers.md)
-  - [Glossary](./concepts/glossary.md)
+  - [Protocol high level overview](./protocol_speficiation/protocol.md)
+  - [Components](/protocol_specification/components.md)
```
Comment (jkrvivian) [inline docs/SUMMARY.md:30]: Maybe put mana and marker as sub-document of Components?
like this:
```suggestion
  - [Components](/protocol_specification/components.md)
     - [Mana]()
     - [Markers]()
```

Comment (daria305) [reply docs/SUMMARY.md:30]: The idea was to not list them in the table of content and just make links clearly visible in components until we have more complete set of documents. But I am no longer sure about this approach

File: docs/protocol_speficiation/components.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+Keeps spammers from harming the network and allocates bandwidth during congested times. 
+
+### Ledger state
+
+The extended Unspent Transaction Output (UTXO) model with the Parallel Reality Ledger State enables nodes to efficiently keep track and update all the balances while managing potential conflicts.
+
+### Consensus
+
+Decentralized random number generator to provide security, fast voting algorithm to provide quick conflict resolutions  and effective finality criteria to fit all applications needs.
```
Comment (daria305) [inline docs/protocol_speficiation/components.md:35]: Maybe we should be less general and mention FPC and FCoB?

File: CHANGELOG.md (hunk 2 above)
Comment (apenzk) [reply CHANGELOG.md:168]: ups

File: docs/SUMMARY.md (hunk 5 above)
Comment (apenzk) [reply docs/SUMMARY.md:30]: I have listed them in the overview list.. lets stick to our initial plan of having them in the protocol overview document

File: docs/protocol_speficiation/components.md (hunk 6 above)
Comment (apenzk) [reply docs/protocol_speficiation/components.md]: ok done

**Review by jkrvivian — APPROVED** [review]

File: plugins/drng/drng.go (hunk 4 above)
Comment (apenzk) [reply plugins/drng/drng.go:16]: ok reverted

**Review by capossele — APPROVED** [review]

**Pull request merged**